### PR TITLE
first version of analysis page.

### PR DESCRIPTION
**Pull request by TS-Stefan**

### CODE REVIEW

File: apps/koala-frontend/src/app/features/sessions/pages/session-analysis/session-analysis.page.ts [hunk 1]
```diff
@@ -1,10 +1,260 @@
...
+import { Marker } from '../../types/marker.entity';
+import { filter } from 'rxjs';
+
+export interface AnnotationData {
+  AnnotationData: Map<number, Array<DataPoint>>;
+}
 
 @Component({
-  selector: 'koala-session-analysis',
+  selector: 'koala-app-session-analysis.page',
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session-analysis/session-analysis.page.ts:21]: what's the reason for the selector change? The pattern in our other components is koala-<name of component without the 'page' suffix> ... is there any check running on this?

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
-  </p-tabView>
+      </p-toolbar>
+      <koala-app-annotation
+        [annotationData]="AnnotationData"
+        [markers]="markers"
+        [totalTime]="totalAudioTime"
+        [currentTime]="currentAudioTime"
+      ></koala-app-annotation>
+      <koala-marker-toolbar
+        [toolbarMode]="ToolbarMode.Session"
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html:18]: ToolbarMode SessionDisabled differentiation was added here with another change. I guess this was removed during rebasing/merging? could you check that?

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
+        [markers]="markers"
+        (event)="onMarkerButtonEvent($event)"
+        [toolbarMode]="ToolbarMode.Session"
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html:21]: Same here, the duplicate of the toolbarMode binding was removed in another change

**Review by TS-Daniel16 — CHANGES_REQUESTED** [review]

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html (hunk 2 above)
Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html]: no your change came later. I rebased it now and fixed the merge conflicts

**Review by TS-Daniel16 — APPROVED** [review]

**Pull request merged**